packages/main/src/RadioButton.ts

```diff
@@ -0,0 +1,536 @@
+import { isDesktop } from "@ui5/webcomponents-base/dist/Device.js";
+import { getFeature } from "@ui5/webcomponents-base/dist/FeaturesRegistry.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import languageAware from "@ui5/webcomponents-base/dist/decorators/languageAware.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import type { ClassMap } from "@ui5/webcomponents-base/dist/types.js";
+import {
+	isSpace,
+	isEnter,
+	isDown,
+	isLeft,
+	isUp,
+	isRight,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import Label from "./Label.js";
+import RadioButtonGroup from "./RadioButtonGroup.js";
+import WrappingType from "./types/WrappingType.js";
+import type { IFormElement } from "./features/InputElementsFormSupport.js";
+import type FormSupport from "./features/InputElementsFormSupport.js";
+
+// Template
+import RadioButtonTemplate from "./generated/templates/RadioButtonTemplate.lit.js";
+
+// i18n
+import {
+	VALUE_STATE_ERROR,
+	VALUE_STATE_WARNING,
+	VALUE_STATE_SUCCESS,
+	VALUE_STATE_INFORMATION,
+	// @ts-ignore
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import radioButtonCss from "./generated/themes/RadioButton.css.js";
+
+let isGlobalHandlerAttached = false;
+let activeRadio: RadioButton|null = null;
```

Comment:
we could remove that null value here. ```let activeRadio: RadioButton;``` will act in the same since we don't reassign the value to null anywhere